schema/schema.go

```diff
@@ -1,86 +1,70 @@
-package schema
+// Code generated by Thrift Compiler (0.17.0). DO NOT EDIT.
```

Comment:
does this mean you used thrift compiler version 0.17?
could this have issues with using 0.14 version of the library?
probably it's backward compatible so should be ok?